apps/explorer/src/components/module/PkgModulesWrapper.tsx

```diff
@@ -0,0 +1,128 @@
+                <Combobox value={selectedModule} onChange={setSelectedModule}>
+                    <div className="box-border border border-sui-grey-50 border-solid rounded-[6px] h-[34px] shadow-sm placeholder-sui-grey-65 pl-3 w-full flex my-[10px] justify-between  ml-[2px]">
+                        <Combobox.Input
+                            onChange={(event) => setQuery(event.target.value)}
+                            displayValue={() => query}
+                            placeholder="Search"
+                            className="border-none w-full"
+                        />
+                        <button
+                            onClick={submitSearch}
+                            className="bg-inherit border-none pr-3"
+                            type="submit"
+                        >
+                            <SearchIcon className="fill-sui-steel cursor-pointer" />
+                        </button>
+                    </div>
+                    <Combobox.Options
+                        static
+                        as="div"
+                        className="w-full h-[209px] md:h-[571px] overflow-auto mr-[-20px] pr-[20px]"
+                    >
+                        <VerticalList>
+                            {filteredModules.map((name, idx) => (
+                                <Combobox.Option key={name} value={name}>
+                                    {({ active }) => (
+                                        <div
+                                            key={idx}
+                                            className="min-w-full w-fit"
+                                        >
+                                            <ListItem
+                                                active={
+                                                    active ||
+                                                    selectedModule === name
+                                                }
+                                            >
+                                                {name}
+                                            </ListItem>
+                                        </div>
+                                    )}
+                                </Combobox.Option>
+                            ))}
+                        </VerticalList>
+                    </Combobox.Options>
+                </Combobox>
+            </div>
+            <div className="border-0 md:border-l border-solid border-sui-grey-45 md:pl-[30px] pt-[20px]">
+                <TabGroup size="md">
+                    <TabList>
+                        <Tab>Bytecode</Tab>
+                    </TabList>
+                    <TabPanels>
+                        <TabPanel>
+                            <div className="overflow-auto h-[555px] w-[87vw] md:w-[70vw] lg:w-[75vw]">
+                                {modules
+                                    .filter(([name]) => name === selectedModule)
```

Comment:
can also use `.find()` to express that there's just one module you're finding here.